graphics/msdfgen/Portfile

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+github.setup        Chlumsky msdfgen 1.9 v
+revision            0
```

Comment:
please add the  "github.tarball_from" lien and set it to "releases" (preferred if upstream provides them) or "archive". This will require most likely  updating the "checksums".

Reply:
#$^*@%, it is starting to get super annoying to have to remember to add this every time. Can we _please_ just make a github 1.1 PortGroup that sets archive to be the default? What are people's objections to doing that?

Reply:
@jasonliu-- You can create that portgroup and do a pull request for it. Meanwhile, could you make the requested changes and then we can commit this?

Reply:
the discussion on the Trac ticket is pretty clear I think: no new PH just because of this. So please do not submit a PR for that; just make the requested change here.

Reply:
Hi @pmetzger , @reneeotten 

I agree that the PortGroup is a separate issue that is being hashed out on the Trac ticket. I caught either COVID or flu last week, and it hit me pretty hard, I'm really only recovering now. In addition, the file server that holds most of my MacPorts-related development stuff died a few days ago, and I am still trying to recover it. As soon as I do, I will add the line to this PR, it will just have to become one of the many ports that will be included in whatever resolution the Trac ticket eventually turns out to be.

Reply:
Okay, I've made the change.

@pmetzger, @reneotten, could one of you take a quick look at PR #20943 as well? The CI is failing on macOS 12/13, but the error doesn't seem to have anything to do with the Portfile itself.